File: modified_scripts/julia/matrix_multiplication/qwen2.5-coder_7b.jl
```julia
function matrix_multiplication(A::Matrix{Float64}, B::Matrix{Float64})::Matrix{Float64}
    C = zeros(size(A, 1), size(B, 2))
    for i in 1:size(A, 1)
        for j in 1:size(B, 2)
            for k in 1:size(A, 2)
                C[i, j] += A[i, k] * B[k, j]
            end
        end
    end
    return C
end
```

File: modified_scripts/r/matrix_multiplication/qwen2.5-coder_7b.r
```r
matrix_multiplication <- function(A, B) {
  n <- nrow(A)
  m <- ncol(B)
  p <- ncol(A)
  
  C <- matrix(0, n, m)
  
  for (i in 1:n) {
    for (j in 1:m) {
      for (k in 1:p) {
        C[i, j] <- C[i, j] + A[i, k] * B[k, j]
      }
    }
  }
  
  return(C)
}
```